scripts/generateProptypes.ts

```diff
@@ -248,7 +248,10 @@ async function generateProptypes(
       ensureBabelPluginTransformReactRemovePropTypesIntegration: true,
       getSortLiteralUnions,
       reconcilePropTypes: (prop, previous, generated) => {
-        const usedCustomValidator = previous !== undefined && !previous.startsWith('PropTypes');
+        const usedCustomValidator =
+          previous !== undefined &&
+          !previous.startsWith('PropTypes') &&
+          !previous.startsWith('refType');
```

Comment:
I would prefer to move this logic inside TTP to share it with X.
For now, if we forget to copy paste on X, we have diverging before for no reason.